src/sailfishusermanager.cpp

```diff
@@ -332,6 +334,25 @@ uint SailfishUserManager::addUser(const QString &name)
     return addSailfishUser(user, name);
 }
 
+void SailfishUserManager::executeScripts(uint uid, const QString &directory)
+{
+    QDir creation(directory, "*.sh", QDir::NoSort, QDir::Files | QDir::Executable);
+
+    auto entryList = creation.entryList();
+
+    QCollator collator;
+    collator.setNumericMode(true);
+    collator.setLocale( QLocale::C );
```

Comment:
To avoid intializing twice, you could set locale already in constructor.
`QCollator collator(QLocale::C);`